### DISCUSSION TITLE
How to check the encryption protocol used when connecting to openvpn

### DISCUSSION THREAD

**Discussion by flowerseacar** [OpenVPN/openvpn3-linux#256] [Q&A - Our support arena] [answered]
My configuration file is configured, but I want to see if the encryption takes effect. There seems to be no encryption protocol information in the log, but no error is reported.
And I can't use wireshark to solve this problem, because openvpn is set up by me to connect through a proxy, and the proxy server is also encrypted. Therefore, it is impossible to judge whether it is encrypted simply based on the egress traffic.

**Comment by dsommers** [comment, marked as answer]
If you enable debug logging and increase the log level in your configuration profile, you will see information like this in your logs:

```
2024-05-02 08:36:24 Client DEBUG: [Core] PROTOCOL OPTIONS:
       cipher: AES-256-GCM
       digest: NONE
       key-derivation: TLS Keying Material Exporter [RFC5705]
       compress: NONE
       peer ID: 1
       control channel: tls-crypt v2 enabled

```

To enable debug logging, as `root` run this command: 

    # openvpn3-admin log-service --log-level 6

To increase the log level in your configuration profiles ... if you have imported the configuration profile using `openvpn3 config-import`, you can do this:

    $ openvpn3 config-manage --show --log-level 6 --config CONFIG_NAME

otherwise add `verb 6` to your configuration file.
